packages/framework-core/src/booster-command-dispatcher.ts

```diff
@@ -43,6 +44,7 @@ export class BoosterCommandDispatcher {
     let result: unknown
     const register: Register = new Register(
       migratedCommandEnvelope.requestID,
+      context.responseHeaders,
```

Comment:
I added the context as a parameter here to allow building the register directly from the context and avoid having the headers object in the `CommandEnvelope`. The previous solution worked, but a command envelope is meant to be a final object because in some implementations it could be useful to be serializable, so it's better that it doesn't contain any mutable state.